webrtc/videoroom/lib/videoroom_web/room_channel.ex

```diff
@@ -1,41 +1,24 @@
-defmodule VideoRoomWeb.RoomChannel do
+defmodule VideoRoomWeb.PeerChannel do
   use Phoenix.Channel
 
   require Logger
 
   @impl true
-  def join("room:" <> room_id, %{"metadata" => metadata}, socket) do
-    # FIXME: make use of structs that are serialized to/from camel case to snake case existing atoms
-
-    # metadata
-
-    room_id =
-      case room_id do
-        "screensharing:" <> id -> id
-        id -> id
-      end
-
-    case VideoRoom.Pipeline.lookup(room_id) do
-      nil -> VideoRoom.Pipeline.start(room_id)
-      pid -> {:ok, pid}
+  def join("room:" <> room_id, _params, socket) do
+    case Registry.lookup(Membrane.Room.Registry, room_id) do
+      [{pid, _value}] -> {:ok, pid}
+      [] -> Membrane.Room.start(name: {:via, Registry, {Membrane.Room.Registry, room_id}})
```

Comment:
This is incompatible with how we name channel of screensharing peer (see `getChannelId` function, declared in `utils.ts` and used in `room.ts`)